openwisp_utils/tests.py

```diff
@@ -76,3 +77,70 @@ def stopTestRun(self):
 class TimeLoggingTestRunner(DiscoverRunner):
     def get_resultclass(self):
         return TimeLoggingTestResult
+
+
+class redirect_stdout(object):
```

Comment:
The name of this class doesn't follow python class naming conventions, however, I have seen some decorators to not follow it either.

@nemesisdesign, do you know if class name conventions are followed for decorator classes or not?